src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -147,6 +148,11 @@ def register_submission_attachment(
             "doc_vertrouwelijkheidaanduiding": "doc_vertrouwelijkheidaanduiding",
         },
     )
+
+    attachment_options["ontvangstdatum"] = (
+        datetime_in_amsterdam(submission.completed_on).date().isoformat()
```

Comment:
I get an error here because of the definition of function `datetime_in_amsterdam` which accepts `datetime` and the definition of `create_document` which accepts `str|None`. Can or should this be fixed?

Reply:
discussed on slack, `assert` statement will help

Reply:
As discussed in slack this will be fixed in the next commit with an assertion (type checker happy  - developers will know that `submission.completed_on` will be set)